tests/wpt/mozilla/meta/mozilla/interfaces.html.ini

```diff
@@ -0,0 +1,4 @@
+[interfaces.html]
+  [Interfaces exposed on the window]
```

Comment:
The interfaces.html and interfaces.worker.js test files should be updated to include the new interfaces instead.

Reply:
Ok